src/repository/screens/read-me.screen.js

```diff
@@ -33,19 +37,40 @@ const styles = StyleSheet.create({
 });
 
 class ReadMe extends Component {
+  static navigationOptions = ({ navigation }) => {
+    const { state } = navigation;
+
+    return {
+      headerRight: (
+        <Icon
+          name="gear"
```

Comment:
Can this icon be better here?

![image](https://user-images.githubusercontent.com/4408379/31176403-6763d4b2-a91b-11e7-9931-f9d27f9cf50c.png)

Reply:
yeah, using ellipsis is more consistent, I will change it

Reply:
On the README screen still use the ellipsis icon.

Reply:
Ops, my bad, sorry 🙈